htdocs/main.php

```diff
@@ -276,12 +276,12 @@ function tplFromRequest($param)
 }
 $tpl_data['userPerms']   = $realPerms;
 $tpl_data['studyParams'] = array(
-                            'useEDC'      => $config->getSetting('useEDC') ?
-        $config->getSetting('useEDC') : false,
-                            'useProband'  => $config->getSetting('useProband') ?
-        $config->getSetting('useProband') : false,
-                            'useFamilyID' => $config->getSetting('useFamilyID') ?
-        $config->getSetting('useFamilyID') : false,
+                            'useEDC'      => $config->getSetting('useEDC'),
```

Comment:
@driusan I removed the ternary check for null, since `getSetting()` either throws an exception of already return the necessary value.